resource_obmcs_identity_policy_test.go

```diff
@@ -28,7 +28,7 @@ func (s *ResourceIdentityPolicyTestSuite) SetupTest() {
 	s.Config = testProviderConfig() + `
 	resource "oci_identity_compartment" "t" {
 		name = "-tf-compartment"
-		description = "tf test compartment"
+		description = "tf test compartment2"
```

Comment:
Is this change required?

Reply:
someone updated the description of compartment, this change was to make the test runnable while I fix the compartment description.
Fixed.